src/components/label/label.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from "@reach/router";
+import React from "react";
+
+const label = ({ title, label, linkTo = "#" }) => (
```

Comment:
I think the only props you need here are `title` (or maybe rather `name`) and `color`, and then it should simply render a `<span>` element with the `name` inside as text, and with `color` as its background-color.